pkg/ibmcsidriver/node.go

```diff
@@ -44,10 +46,11 @@ import (
 
 // CSINodeServer ...
 type CSINodeServer struct {
-	Driver   *IBMCSIDriver
-	Mounter  mountmanager.Mounter
-	Metadata nodeMetadata.NodeMetadata
-	Stats    StatsUtils
+	Driver             *IBMCSIDriver
+	Mounter            mountmanager.Mounter
```

Comment:
with this code changes `Mounter` will not be useful, I see we can utilise `Mounter` by implementing this PR logic in https://github.com/IBM/ibm-csi-common repo

@ambiknai , can you check the possibility for this so that we will have a mounter as a package with interface

Reply:
@arashad @ambiknai thank you for cooperation, I believe we're done with the mounter changes.